ui/src/components/microsegmentation/AddSegmentation.js

```diff
@@ -324,6 +406,56 @@ export default class AddSegmentation extends React.Component {
             });
     }
 
+    validatePort(port) {
+        var regex = new RegExp('^[0-9-,]*$');
+        var result = {
+            error: 0,
+            port: port,
+        };
+
+        if (!regex.test(port)) {
+            this.setState({
+                errorMessage: "Port can only contain numbers, '-' and ','",
+            });
+            result.error = 1;
+            return result;
+        }
+
+        let ports = port.split(',');
+        for (var i = 0; i < ports.length; i++) {
+            if (ports[i].indexOf('-') != -1) {
+                let range = ports[i].split('-');
+                let start = parseInt(range[0]);
+                let end = parseInt(range[1]);
+                if (
+                    range.length != 2 ||
+                    start < 1 ||
+                    end < 1024 ||
+                    start > end ||
+                    start > 65535 ||
+                    end > 65535
+                ) {
+                    this.setState({
+                        errorMessage: 'Invalid port: ' + ports[i],
+                    });
+                    result.error = 1;
+                    return result;
+                }
+            } else if (ports[i] > 0 && ports[i] <= 65535) {
+                ports[i] = ports[i] + '-' + ports[i];
+            } else {
+                this.setState({
+                    errorMessage: 'Port has to be in the range of 0-65535',
```

Comment:
if we are enforcing start >=1 ( in the if condition above ) then lets use 1-65535